examples/even-odd-list-with-type-level-booleans

```diff
@@ -0,0 +1,45 @@
+/// A list of type `a` that uses a Boolean at the type level to track
```

Comment:
Filename: `type-level-programming-odd-even-list.flix`

Reply:
smh I initially changed it to `type-level-programming-odd-even-list.filx`
Should be in accordance with all of your comments now.